addons/debuginfo/debuginfo.lua

```diff
@@ -0,0 +1,59 @@
+_addon.name = 'debuginfo'
+_addon.author = 'Lili'
+_addon.version = '1'
+_addon.command = 'debuginfo'
+
+require('logger')
+
+local       get_dir = windower.get_dir
+local windower_path = windower.windower_path
+local     ffxi_path = windower.ffxi_path
+local      pol_path = windower.pol_path
+-- local launcher_path = windower.launcher_path 
+
+local dlls = { 'd3d8.dll', 'dxgi.dll', 'ddraw.dll', 'd3dimm.dll', 'd3d9.dll', '.*%.conf' }
+
+windower.register_event('addon command',function(arg)
+    local arg = arg:lower()
+
+    local windower_settings = windower.get_windower_settings()
+
+    local str = ''
+
+    str = str:append('\nWindower path: ' .. windower_path)
+    str = str:append('\n Launcher version: ' .. windower_settings.launcher_version)
+    str = str:append('\n Hook version: ' .. windower_settings.hook_version)
+    str = str:append('\nFFXI path: ' .. ffxi_path)
+    str = str:append('\n Client version: ' .. windower_settings.ffxi_version)
+
+    -- Look for cases of mismatched foldername\addonname.lua
+    local addons_path = windower_path .. 'addons\\'
+    local addons = get_dir(addons_path)
+    for _,name in pairs(addons) do
+        if name:endswith('-master') then
```

Comment:
I change this check to this:
```lua
if name ~= 'libs' then
```

That's because the following check is *always* valid and can find misnamed directories even outside of the `-master` issue. If a directory with name `<name>` exists, it *must* contain a `<name>.lua` inside, otherwise it's not a valid addon (unless it's the `libs` directory).

Reply:
Implemented.